src/components/pages/Documentation/Markdown/ToggleProvider/index.tsx

```diff
@@ -1,11 +1,19 @@
 import React, { createContext, useEffect, useState } from 'react'
 
 interface ITogglesData {
-  [key: string]: { texts: string[]; checkedInd: number }
+  [key: string]: {
+    texts: string[]
+    checkedInd: number
+    parentText: string | null
```

Comment:
For now tabs can only have one parent. So you can't create a  nested tab set inside a nested tab. 

```
<toggle>
<tab>

I'm the top level tab!

<toggle>
<tab>

I'm nested tab! this works fine

<toggle>
<tab>

I'm a nested tab inside a nested tab. this will not work

</tab>
</toggle>

</tab>
</toggle>

</tab>
</toggle>
```

If this gets merged, I'll open an issue for allowing nested tabs go past one level.

Reply:
I'm not sure we need more levels atm, so not worth bothering (unless easy to do).